### DISCUSSION TITLE
Diary-最后一个暑假 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#344] [Announcements]
# Diary-最后一个暑假 | Zi-Zi's Journey

我知道 那些夏天 就像青春一样回不来

https://gz-metal-cell.github.io/2024/08/23/Diary-%E6%9C%80%E5%90%8E%E4%B8%80%E4%B8%AA%E6%9A%91%E5%81%87/

**Comment by promefire** [comment]
太强了，希望出教程

**Reply by GZ-Metal-Cell** [reply]
前端把戏 都是些没用的😭

**Reply by promefire** [reply]
快一年了关神什么时候出教程😭

**Reply by GZ-Metal-Cell** [reply]
翻页是 turn.js，下面的日历是 EChart，照着官方文档给的 API 慢慢敲😭
